service/javadsl/integration-tests/src/test/java/com/lightbend/lagom/it/none/NoServiceTest.java

```diff
@@ -22,8 +24,9 @@ public void testAModuleWithBrokerConsumptionButWithoutServiceCanBeRun() throws E
         Setup setup = defaultSetup()
                 .withCluster(false)
                 .configureBuilder(b ->
-                        b.bindings(new NoServiceModule()).overrides(
-                                bind(PublisherService.class).to(PublisherServiceStub.class))
+                        b.bindings(new NoServiceModule())
+                            .overrides(EmptyAdditionalRoutersModule.instance())
```

Comment:
I had to add this `EmptyAdditionalRoutersModule` because some of our tests are not based on `ServiceGuiceSupport` and binds the Lagom router directly. Without this, Guice fails to bind it because of missing `List<Router>` binding. 

When using `ServiceGuiceSupport` we don't need to have this extra empty binding.